openedx/openedx-platform
# Remove tagging object rest api pagination

## Description

Removes the pagination of the tagging REST API for listing and updating object tags.

## Supporting Information

* Closes https://github.com/openedx/modular-learning/issues/99
* Depends on:
  * https://github.com/openedx/openedx-learning/pull/81/
  * https://github.com/openedx/edx-platform/pull/33259

## Testing instructions
* Please ensure that the tests cover the expected behavior of the view as described in the related issue.

____
Private-ref: [FAL-3501](https://tasks.opencraft.com/browse/FAL-3501)
